<launch>
    <node pkg="tf" type="static_transform_publisher" name="transform_sonar_frame" args="0 0 0 0 0 0 1 map sonar_frame 10" />
    <env name="emulated_sonar" value="false" />
    <node pkg="ping360_sonar" type="ping360_node" name="ping360_node" output="screen">
        <param name="device" value="/dev/ttyUSB0"/>
        <param name="baudrate" value="115200"/>
        <param name="debug" value="False"/>
        <param name="imgSize" value="500"/>
        <param name="gain" value="0"/>
        <param name="step" value="1"/>
        <param name="numberOfSamples" value="200"/>
        <param name="transmitFrequency" value="740"/>
        <param name="sonarRange" value="10"/>
        <param name="speedOfSound" value="1500"/>
        <param name="queueSize" value="1"/>
        <param name="threshold" value="100"/>
        <param name="enableImageTopic" value="True"/> 
        <param name="enableScanTopic" value="True"/> 
        <param name="enableDataTopic" value="True"/> 
        <param name="maxAngle" value="400"/> 
    </node>
</launch>